commands/displayers/database.go

```diff
@@ -155,6 +155,36 @@ func (db *DatabaseBackups) KV() []map[string]any {
 	return out
 }
 
+type DatabaseCA struct {
+	DatabaseCA do.DatabaseCA
+}
+
+var _ Displayable = &DatabaseCA{}
+
+func (dc *DatabaseCA) JSON(out io.Writer) error {
+	return writeJSON(dc.DatabaseCA, out)
+}
+
+func (dc *DatabaseCA) Cols() []string {
+	return []string{
+		"Certificate",
+	}
+}
+
+func (dc *DatabaseCA) ColMap() map[string]string {
+	return map[string]string{
+		"Certificate": "Certificate",
+	}
+}
+
+func (dc *DatabaseCA) KV() []map[string]any {
+	return []map[string]any{
+		{
+			"Certificate": string(dc.DatabaseCA.Certificate),
```

Comment:
`json` and `text` formats have different certificate values. 
users won't be able to connect with the cert taken from json :crying_cat_face:

Reply:
The value is base64 encoded. As that matches the API response, I think that is OK. Though we should probably mention it in the help output for the command.

This should show the same value as the text output:

```
./builds/doctl databases get-ca <uuid> -o json | jq -r .certificate | base64 --decode
```

Reply:
Thanks a lot, Andrew :pray: This really helps :+1: 

I'm struggling to explain it in a simple way to our users, but I'll try my best :slightly_smiling_face: 
I'm afraid that this difference can be confusing and will lead to more open issues down the road as people may expect json and text to use the same encoding...